spec/controllers/tags_controller_spec.rb

```diff
@@ -4,7 +4,29 @@
   include LoginMacros
   include RedirectExpectationHelper
 
-  before { fake_login_known_user(create(:tag_wrangler)) }
+  let(:user) { create(:tag_wrangler) }
+
+  before { fake_login_known_user(user) }
+
+  shared_examples "set last wrangling activity" do
+    it "sets the last wrangling activity time to now" do
+      expect(user.last_wrangling_activity.updated_at).to be_within(1.minute).of Time.now.utc
+    end
+  end
+
+  describe "#create" do
+    let(:tag_params) do
+      { name: Faker::FunnyName.name, canonical: "0", type: "Character" }
+    end
+
+    before { freeze_time }
```

Comment:
For the tests with shared example [`"set last wrangling activity"`](https://github.com/otwcode/otwarchive/pull/4129/files/e45ccd7dd8750eefa0411e9de1ddb3d09f60e1d9#diff-e1135e966d22b227e95c7a9676f4ad1384cd8e1a09920aa74e9c451d0c88a6fbR11), I initially had a check for last wrangling time being equal to now. Somewhere along the line I switch to just check that timestamp is within the last minute -- I think because there was a weird precision thing somewhere -- but never removed these. I've done that now, but if you'd prefer I track the precision thing down and use `freeze_time` again I can also do that